cli/src/main/kotlin/Extensions.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.cli
+
+/**
+ * Add a trailing slash to this [String] if it is missing.
+ */
+fun String.ensureTrailingSlash(): String = takeIf { endsWith("/") } ?: "$this/"
```

Comment:
Funnily enough, as ORT already was `fun String?.withoutSuffix()`, I was considering to add `ensureSuffix(suffix: String)`. If that comes, we could replace this again later on. Or already make this also more generic by taking the desired suffix as an argument.